snapshot/btrfs/btrfs_test.go

```diff
@@ -1,4 +1,4 @@
-// +build linux
+// +build linux,!no_btrfs
```

Comment:
All you need is this change to prevent tests from running.  Why did you change the other two places? We don't want to sub out an entire package, just the import that we are already doing in the containerd.main